tests/test_edgeql_functions.py

```diff
@@ -2099,6 +2099,22 @@ async def test_edgeql_functions_to_str_08(self):
             [''],
         )
 
+    async def test_edgeql_functions_to_array_01(self):
+        await self.assert_query_result(
+            r'''SELECT to_array('one, two, three', ', ');''',
+            [['one', 'two', 'three']],
+        )
+
+        await self.assert_query_result(
+            r'''SELECT to_array('', ', ');''',
+            [[]],
+        )
+
+        await self.assert_query_result(
+            r'''SELECT to_array('foo', ', ');''',
+            [['foo']],
```

Comment:
What happens if the delimiter is an empty string? Please add a test (and document).

Reply:
Done.